j2v/str_templates/sql_templates.py

```diff
@@ -1,2 +1,4 @@
 join_str_template = "LATERAL FLATTEN(OUTER => TRUE, INPUT => {exploded_structure_path}) {alias}"
 field_str_template = "{TABLE}.{__path}::{json_type} AS {path_alias}"
+non_nullable_text_field_str_template = "IFNULL({TABLE}.{__path}::{json_type},'N/A') AS {path_alias}"
```

Comment:
I think we should have a generic template for null fields, and have <expr2> ('N/A' OR 0) as parameters to the program. I can imagine many other popular replacements for NULL strings, such as - or a space bar. The same for numeric values.